api-server/common/models/user.js

```diff
@@ -758,12 +758,33 @@ export default function(User) {
     });
   };
 
+  function parseDate(joinDate) {
+    const monthNames = [
+      'January',
+      'February',
+      'March',
+      'April',
+      'May',
+      'June',
+      'July',
+      'August',
+      'September',
+      'October',
+      'November',
+      'December'
+    ];
+    const year = joinDate.getFullYear();
+    const month = monthNames[joinDate.getMonth()];
+    return 'Joined freeCodeCamp in ' + month + ' of ' + year;
+  }
```

Comment:
This could be done on client instead, and keep the API pure.

Reply:
Thank You for the review! I'll move the processing logic to the client.